src/components/itemHelper.js

```diff
@@ -304,14 +284,11 @@ export function supportsMediaSourceSelection (item) {
     if (!item.MediaSources || (item.MediaSources.length === 1 && item.MediaSources[0].Type === 'Placeholder')) {
         return false;
     }
-    if (item.EnableMediaSourceDisplay === false) {
-        return false;
-    }
-    if (item.EnableMediaSourceDisplay == null && item.SourceType && item.SourceType !== 'Library') {
+    if (!item.EnableMediaSourceDisplay) {
         return false;
     }
 
-    return true;
+    return !item.SourceType || item.SourceType === 'Library';
```

Comment:
Looks like it was equal to:
```js
if (item.EnableMediaSourceDisplay != null) {
    return !!item.EnableMediaSourceDisplay;
}

return !item.SourceType || item.SourceType === 'Library';
```
So, you were able to force "MediaSourceSelection" support.

Not sure which is the right one.

Reply:
There was also a check for `EnableMediaSourceDisplay` being false above though.

Reply:
Yes, the equivalent includes it. If `EnableMediaSourceDisplay` explicitly `false`, return `false`. If it is undefined or null, check `item.SourceType`. Otherwise (`EnableMediaSourceDisplay` is defined and is not `false` and `item.SourceType` is undefined or is a library), return `true`.
_It probably didn't work correctly for `EnableMediaSourceDisplay: 0` (!== false)._